### ISSUE TITLE
нулевой 1sg.pos

### ISSUE THREAD

**Issue by veramal** [tuffnatty/suddenly#61]
Вот еще интересно про посессивность - в 1sg у слов на _м_ может опускаться:

ТААМ цель, заветное желание; 
_минің **таам** андағ_ - таково моё желание; 
_паланың **таамын** піл полбинчалар_  - [они] не могут понять, чего хочет ребёнок

Для парсера это неважно, но вообще интересно.

**Comment by adybo** [comment]
Да, здорово. Если это только на -м, то опускается, а если не только, то это конструкция, которую традиционно называют "изафет второго типа"

**Comment by veramal** [comment]
Изафет второго типа все же устроен наоборот: неоформленный атрибут+вершина с посессивностью.
А пример про желание ребенка лишний.
